defaults/main.yml

```diff
@@ -34,3 +35,5 @@ postfix_mynetworks:
 postfix_smtpd_banner: '$myhostname ESMTP $mail_name (Ubuntu)'
 postfix_disable_vrfy_command: false
 postfix_message_size_limit: 10240000
+postifx_header_check_format: regexp
```

Comment:
Rename to `postfix_header_checks_database_type`

Reply:
done in newest commit